ci/script.sh

```diff
@@ -1,14 +1,29 @@
 #!/usr/bin/env bash
 
+source "./ci/lib/travis_helpers.sh"
+
 set -euo pipefail
 
+# Executing build actions
+echo "Executing the build actions: $BUILD_ACTIONS"
 xcrun xcodebuild $BUILD_ACTIONS \
-	NSUnbufferedIO=YES \
+    NSUnbufferedIO=YES \
     -project SPTDataLoader.xcodeproj \
     -scheme "$SCHEME" \
     -sdk "$TEST_SDK" \
     -destination "$TEST_DEST" \
     $EXTRA_ARGUMENTS \
-         | xcpretty -c -f `xcpretty-travis-formatter`
+        | xcpretty -c -f `xcpretty-travis-formatter`
+
+
+# License conformance
+travis_fold_open "License conformance"
+./ci/validate_license_conformance.sh {include/SPTDataLoader/*.h,SPTDataLoader/*.{h,m}}
```

Comment:
I put the license validation after the general build actions. However as they run so fast, and they will fail the build, should we have the check before building and test? Wondering if that would make the fail/fix cycle quicker.

Downside to that is that you’d have to fix the license issue (trivial) before you’re even told about potential code failures.